src/Factories/ProcessFactory.php

```diff
@@ -53,4 +53,25 @@ public function createCyclomaticComplexityProcess(File $file): ChurnProcess
 
         return new ChurnProcess($file, $process, 'CyclomaticComplexityProcess');
     }
+
+    private function getVscCommand(File $file): string
+    {
+        $path = $file->getFullPath();
+        while ($dir = dirname($path)) {
+            if ($dir == $path) {
+                break;
+            }
+
+            if (is_dir($dir . '/.git')) {
+                return 'git log --since="' . $this->config->getCommitsSince() . '"  --name-only --pretty=format: ' . $file->getFullPath();
+            } elseif (is_dir($dir . '/.hg')) {
```

Comment:
Related to the other comment about having a separate clss for each process. Otherwise if this project ends up supporting multiple VCS this if/else will keep growing which I wouldn't want.